.github/workflows/ci_workflows.yml

```diff
@@ -72,6 +72,13 @@ jobs:
             toxposargs: --remote-data --run-slow
             allow_failure: true
 
+          - name: Python 3.11 with latest dev versions, and Roman-specific dependencies
+            os: ubuntu-latest
+            python: '3.11'
+            toxenv: py311-test-devdeps-romandeps
```

Comment:
I don't like adding roman tests in the CI like this. It is probably going to fail a lot at this stage, and that will make reviewing unrelated PRs a pain.

Reply:
I set `allow_failure: true` with this in mind. Should we leave the `romandeps` option in there, but not add it to the CI workflow?

Reply:
I think I need to understand the plan of this PR and timeline of merge first before I can answer.